crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -173,28 +173,25 @@ impl Variable {
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 pub enum ObjectType {
-    NativeField,
     Boolean,
-    Unsigned(u32), //bit size
-    Signed(u32),   //bit size
+    // TODO: Why don't we put Boolean under Numeric type (u1)?
```

Comment:
Might as well include the bool change as part of this PR

Reply:
Done